### ISSUE TITLE
Plugin project from official npx template fails to run test harness due to some grafana toolkit config file missing

### ISSUE THREAD

**Issue by lweberk** [grafana/grafana#38561]
**What happened**:

npm run test (internally `grafana-toolkit plugin:test`) throws an unexpected error:

```
● Validation Error:

  Module /home/user/path/to/plugin/dir/testplugin-panel/node_modules/@grafana/toolkit/src/config/matchMedia.js in the setupFiles option was not found.
         <rootDir> is: /home/user/path/to/plugin/dir/testplugin-panel

  Configuration Documentation:
  https://jestjs.io/docs/configuration
```

**What you expected to happen**:

For the Jester tests to run and output the results.

**How to reproduce it (as minimally and precisely as possible)**:

Create project directory from template as instructed in official grafana plugin dev documentation;

```bash
npx @grafana/toolkit plugin:create testplugin-panel
```

Optionally add some tests or just straight run the template provided `module.test.ts`.

```bash
npm run test
```
**Anything else we need to know?**:

It can be worked around by editing top-level `jest.config.js` to remove the missing file from the list of `setupFiles` (commented out in this case);

```js
// This file is needed because it is used by vscode and other tools that
// call `jest` directly.  However, unless you are doing anything special
// do not edit this file

const standard = require('@grafana/toolkit/src/config/jest.plugin.config');

// console.log(`${JSON.stringify(standard.jestConfig(), null, 4)}`);

let config = standard.jestConfig();
config.setupFiles = [
    // "/home/user/path/to/plugin/dir/testplugin-panel/node_modules/@grafana/toolkit/src/config/matchMedia.js",
    "jest-canvas-mock"
];

// This process will use the same config that `yarn test` is using
module.exports = config;
```

package.json contents;

```
{
  "name": "testplugin-panel",
  "version": "1.0.0",
  "description": "Reproducing Bug",
  "scripts": {
    "build": "grafana-toolkit plugin:build",
    "test": "grafana-toolkit plugin:test",
    "dev": "grafana-toolkit plugin:dev",
    "watch": "grafana-toolkit plugin:dev --watch",
    "sign": "grafana-toolkit plugin:sign",
    "start": "yarn watch"
  },
  "author": "lweberk",
  "license": "Apache-2.0",
  "devDependencies": {
    "@grafana/data": "latest",
    "@grafana/toolkit": "latest",
    "@grafana/ui": "latest",
    "emotion": "10.0.27"
  },
  "engines": {
    "node": ">=14"
  }
}
```

**Environment**:
- Grafana version:  8.1.1
- User OS & Browser: Debian 11.x

**Comment by jackw** [comment]
@lweberk it looks like [a fix](https://github.com/grafana/grafana/pull/38694) for this issue was merged recently and should be available in 8.1.3+. Could you try updating your plugin `@grafana` dependencies to see if it fixes this issue?

**Comment by lweberk** [comment]
@jackw confirmed fix for version 8.1.5 on own project and also tested with a blank template drawn first as 8.1.5 and then downgraded to 8.1.3 to confirm lowest working version threshold you provided.

My appreciation and gratitude to you all for your work.
